project-base/storefront/components/Pages/Order/TransportAndPayment/TransportAndPaymentSelect/TransportAndPaymentSelect.tsx

```diff
@@ -309,3 +201,13 @@ const ResetButton: FC<ResetButtonProps> = ({ text, dataTestId, onClick }) => (
         <ArrowIcon className="ml-2" />
     </button>
 );
+
+const getPickupPlaceDetail = (
```

Comment:
Maybe better place would be in `helpers.tsx`?

Reply:
Hmmm... could be. I moved it. Let's see if we like it or not 👍